src/test/java/seedu/address/storage/JsonAdaptedClientTest.java

```diff
@@ -2,25 +2,32 @@
 
```

Comment:
Not sure why the formatting in this file changed so much also, would suggest to change ur autoformatter settings to match the project's code style, or not use the autoformatter, if not unrelated lines will keep changing in your commits

Reply:
^

Reply:
updated